lib/gds_api/test_helpers/content_api/artefact_stub.rb

```diff
@@ -38,19 +38,20 @@ def stub
               .with(query: hash_including(comparable_query_params))
               .to_return(status: @response_status, body: @response_body.to_json)
         end
-        
-        private
-          def url_without_query
-            "#{CONTENT_API_ENDPOINT}/#{CGI.escape(slug)}.json"
-          end
 
-          # Ensure that all keys and values are strings 
+      private
+
+        def url_without_query
+          "#{CONTENT_API_ENDPOINT}/#{CGI.escape(slug)}.json"
+        end
+
+          # Ensure that all keys and values are strings
```

Comment:
It might be the way github is showing the diff, but it looks like this comment is overindented now.